### DISCUSSION TITLE
"Search" for Invoice based Loans

### DISCUSSION THREAD

**Discussion by SanjanaIL** [ONDC-Official/ONDC-FIS-Specifications#37] [General]
1) Will the Search API filter out which Lenders are willing to lend to the User or all Lenders registered on the network be shown?
2) On the basis on what information will the lender decide what users is it willing to lend to?
3) How are we making sure that relevant lenders are being shown to the customers on the buyer app so that high approval % can be achieved?

**Comment by 92shreyansh** [comment]
As part of the search API there is no customer info that is transmitted to the lender, All Lenders will receive the search request. The forms received part of on_search will capture the customer info. This form is submitted and linked submission id is sent via select as part of this lender can choose if or not to respond with a offer or not.

**Reply by madhus1025** [reply]
The form link which is sent as part of the on_search api would be opened in a new window, from the buyers page current window we can't access that form window as that would be CORS violation, in this case how can we get the submission_id ?
